src/foam/nanos/crunch/UserCapabilityJunctionDAO.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'UserCapabilityJunctionDAO',
+  extends: 'foam.dao.ProxyDAO',
```

Comment:
Would it be simpler to just make the UserCapabilityJunction be Authorizable and then just use the AuthorizationDAO?

Reply:
i think the decorator should stay because it's not only doing authorization but also validating data and prereqs